sympy/core/relational.py

```diff
@@ -237,6 +237,25 @@ def canonical(self):
 
         return r
 
+    def applyfunc(self, op):
+        """Apply the same function to both sides of the equation.
+        Calls the function for each side, passing the lhs/rhs as
+        an argument, and then returns a new equation consisting of
+        the returned values."""
+        return self.__class__(op(self.lhs), op(self.rhs))
```

Comment:
I would prefer `self.func` to `self.__class__` because that it how `Basic` is supposed to work in general.

Reply:
Good point, fixed.